solutions/standard/DA-cbr_rules.md

```diff
@@ -0,0 +1,61 @@
+# Configuring complex inputs for KMS in IBM Cloud projects
+
+Several optional input variables in the IBM Cloud [KMS all inclusive deployable architecture](https://cloud.ibm.com/catalog#deployable_architecture) use complex object types. You specify these inputs when you configure deployable architecture.
+
+* Context-Based Restrictions Rules (`cbr_rules`)
+
+
+## Rules For Context-Based Restrictions <a name="cbr_rules"></a>
+
+The `cbr_rules` input variable allows you to provide a rule for the target service, in this case the KMS all inclusive DA instance, to enforce access restrictions for the instance based on the context of access requests. Context are criteria that include the network location of access requests, the endpoint type from where the request is sent, etc.
```

Comment:
`in this case the KMS all inclusive DA instance` - its not correct. KMS all inclusive is just the name of the repo not the DA name. Also, we should not call is as DA instance.

Can we say:
`The `cbr_rules` input variable allows you to provide a rule for the target service to enforce access restrictions for the service based on the context of access requests.`

Reply:
`Context are criteria` --> 
either `Contexts are criteria` OR `Context is criteria`

Please check the Cloud docs to be consistent in the statements and then update

Reply:
done https://github.com/terraform-ibm-modules/terraform-ibm-kms-all-inclusive/pull/594/commits/67a2aa1c5119d9bbaed014260a89ecfed7c12f95